fms2_io/netcdf_io.F90

```diff
@@ -1576,18 +1578,20 @@ function get_valid(fileobj, variable_name) &
       endif
     endif
     valid%has_range = has_min .and. has_max
-  endif
+  endif !if (fileobj%is_root)
 
-  call mpp_broadcast(valid%has_range, fileobj%io_root, pelist=fileobj%pelist)
-  if (valid%has_range) then
+  if (valid%has_range) then !If it found has_range broadcast to the rest of the pes
```

Comment:
@wrongkindofdoctor @uramirez8707 
I don't think this will work.  The broadcasts of valid%has_range, valid%has_fill, and valid%has_missing must
happen before the if statements or I think the code can hang (when the root has the value of .true. for one of these flags).